tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/resolver/TestInputResolver.java

```diff
@@ -11,9 +11,12 @@
 import org.mule.runtime.api.metadata.MetadataContext;
 import org.mule.runtime.api.metadata.MetadataResolvingException;
 import org.mule.runtime.api.metadata.resolving.InputTypeResolver;
+import org.mule.runtime.api.metadata.resolving.MetadataComponent;
 
 public class TestInputResolver implements InputTypeResolver<String> {
 
+  public static String INPUT_RESOLVER_NAME = MetadataComponent.INPUT.name();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make INPUT_RESOLVER_NAME a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7668&issues=AWjdfEJ1otIXnmsUGfbK&open=AWjdfEJ1otIXnmsUGfbK)

[ISSUE_KEY]: AWjdfEJ1otIXnmsUGfbK